assembl/static2/package.json

```diff
@@ -66,6 +66,7 @@
     "json-loader": "^0.5.4",
     "lodash": "^4.17.4",
     "moment": "^2.18.1",
+    "piwik-react-router": "assembl/piwik-react-router#skip_piwik_initialization",
```

Comment:
Interesting choice. By doing this, we have eliminated part of the analytics lib. It was meant to be a general purpose tool, and it included code on setting up multiple analytics engines on a page, if you so wished. I understand the need (to connect this analytics engine to react-router), I wouldn't recommend extending our own library to support react-router anyway.